app/views/users/show.html.erb

```diff
@@ -1,2 +1,20 @@
-<h1>Users#show</h1>
-<p>Find me in app/views/users/show.html.erb</p>
+<div class="user">
+  <div class="profile-image">
+  </div>
+  <div class="user-info">
+    <p class="fs-5"><%= @user.Name %></p>
+    <div class="num-posts">
+      <p>Number of posts: <%= !@user.PostsCounter ? 0 : @user.PostsCounter %> </p>
+    </div>
+  </div>
+</div>
+<div class="bio">
+  <p class="fs-4">Bio</p>
+  <p><%= @user.Bio %></p>
+</div>
+<% @user.recent_posts.each do |post| %>
+<div class="post">
+  <p class="fs-5">Post <%= post.id %> </p>
+  <p><%= post.Text %> </p>
+</div>

```

Comment:
- [ ] On the user page you should display only the 3 most recent posts and then here you forgot to add 'see all post' link to be able to see all user posts, please add that

Reply:
Hi @DeliceLydia, The method that I used here `recent_posts` shows only 3 posts.

Reply:
Ok I will add `see all posts` button